frontend/public/index.html

```diff
@@ -28,7 +30,7 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <div id="root" class="site-wrapper"></div>
```

Comment:
dlaczego dodales tutaj classe?

Reply:
to bylo mi potrzebne do sticky footer, ewentualnie moge dac jakis dodatkowy wrapper w srodku